map_model/src/road.rs

```diff
@@ -113,11 +108,30 @@ pub struct Road {
     pub dst_i: IntersectionID,
 }
 
+type HomogenousTuple2<T> = (T, T);
+
 impl Road {
-    pub fn get_lane_types(&self) -> (Vec<LaneType>, Vec<LaneType>) {
+    pub fn children(&self, fwds: bool) -> &Vec<(LaneID, LaneType)> {
+        if fwds {
+            &self.children_forwards
+        } else {
+            &self.children_backwards
+        }
+    }
+
+    pub fn children_mut(&mut self, fwds: bool) -> &mut Vec<(LaneID, LaneType)> {
```

Comment:
`pub(crate)` please -- nothing outside of `map_model` should reach in and change things